bucket/x64dbg.json

```diff
@@ -43,10 +43,10 @@
         "release\\x64\\plugins"
     ],
     "checkver": {
-        "github": "https://github.com/x64dbg/x64dbg",
+        "github": "https://sourceforge.net/projects/x64dbg/rss?path=/",
```

Comment:
Sourceforge is not GitHub

You will have to ue URL there

Reply:
I fixed the key name.